docs/user-guide.md

```diff
@@ -109,18 +110,45 @@ Initial steps similar to those of the Single Instance installation.
 
    `./install-oracle.sh --help`
 
-1. Run installation. Only IP address of the first cluster node is required:
+1. Create cluster config file. The file `cluster_config.json` is provided as template
```

Comment:
For grammar and brevity, I recommend revising this line to something like:
"Create the cluster configuration file by editing the cluster_config.json file template that is provided with the toolkit."

Reply:
I made the change